Remove note callouts from the docs (#7433)
segmentio/segment-docs: src/connections/destinations/catalog/recombee-ai/index.md

    link: /docs/connections/destinations/catalog/actions-recombee
---

> <<<<<<< Updated upstream

Author: ```suggestion
```

[Recombee](https://recombee.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} is a Recommender as a Service that can use your data to provide the most accurate recommendations of content or products for your users.

> =======

Author: ```suggestion
```

---

[Recombee](https://recombee.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} is a Recommender as a Service that can use your data to provide the most accurate recommendations of content or products for your users.

> Use this Segment destination to send your interaction data views, purchases, plays, etc.) to Recombee.

Reviewer: ```suggestion
Use this Segment destination to send your interaction data (views, purchases, plays, and so on) to Recombee.
```
